observatory-dags/observatory/dags/database/schema/grid_2015-09-22.json

```diff
@@ -1,139 +1,166 @@
 [
   {
+    "description": "The ID of the institute which should now be used",
     "mode": "NULLABLE", 
     "name": "redirect", 
     "type": "STRING"
   }, 
   {
     "fields": [
       {
+        "description": "The unique ID of the related institute",
         "mode": "NULLABLE", 
         "name": "id", 
         "type": "STRING"
       }, 
       {
+        "description": "The name of the related institute",
         "mode": "NULLABLE", 
         "name": "label", 
         "type": "STRING"
       }, 
       {
+        "description": "The relationship type, see our for more details on the types and the meaning of these",
```

Comment:
Would you be able to provide a URL to the page that they refer to that lists the relationship types?